src/test/java/org/snomed/snowstorm/extension/ExtensionAdditionalLanguageRefsetUpgradeServiceTest.java

```diff
@@ -300,8 +301,8 @@ void testGenerateAdditionalLanguageRefsetsWithDeltaOnly_shouldNotUpdateOrAddNewA
 		assertEquals(1, published.size());
 		assertEquals(nzExistingMemberId, published.get(0).getMemberId());
 
-		assertEquals(1, updatedResult.get().filter(referenceSetMember -> referenceSetMember.getReferencedComponentId().equals("2156578010")).collect(Collectors.toList()).size());
-		assertEquals(1, updatedResult.get().filter(referenceSetMember -> referenceSetMember.getReferencedComponentId().equals("675173021")).collect(Collectors.toList()).size());
-		assertEquals(0, updatedResult.get().filter(referenceSetMember -> referenceSetMember.getReferencedComponentId().equals("675173020")).collect(Collectors.toList()).size());
+		assertEquals(1, updatedResult.get().filter(referenceSetMember -> referenceSetMember.getReferencedComponentId().equals("268524019")).collect(Collectors.toList()).size());
+		assertEquals(1, updatedResult.get().filter(referenceSetMember -> referenceSetMember.getReferencedComponentId().equals("588524019")).collect(Collectors.toList()).size());
+		assertEquals(0, updatedResult.get().filter(referenceSetMember -> referenceSetMember.getReferencedComponentId().equals("288524019")).collect(Collectors.toList()).size());
```

Comment:
@QuyenLy87 Can you add an additional test case please? During current extension upgrade, a published PT in extension English language refset is changed to acceptable after adding  a new PT in core module. After upgrade both changes should be applied in extension default English language refset too.